lib/active_admin/filters/active_filter.rb

```diff
@@ -26,19 +26,29 @@ def values
       end
 
       def label
+        translated_predicate = translated_predicate_name.mb_chars.downcase.to_s
```

Comment:
Yes, your solution works in every ruby we support.

I'm fine with it, I would maybe add a `TODO:` to remind us to go back to the simpler `str.downcase` once we support `ruby >= 2.4` only.